src/services/dispatchers.ts

```diff
@@ -94,18 +96,18 @@ async function extractDispatchersFromFile(path: string): Promise<DispatcherCreat
 function assertRequiredFields(dispatcher: DispatcherCreation) {
   // Ensure all dispatchers have names
   if (!dispatcher.name) {
-    throw new Error('Dispatcher name is a required field');
+    throw new Error('Invalid Dispatcher: Dispatcher without a name');
   }
 
   const hasActions = Array.isArray(dispatcher.actions) && dispatcher.actions.length > 0;
   if (!hasActions) {
-    throw new Error('A Dispatcher must have at least one action');
+    throw new Error(`Invalid Dispatcher: ${dispatcher.name} needs at least one action`);
   }
 
   // Ensure all actions have names
   const hasValidActions = dispatcher.actions.every(action => action.name);
   if (!hasValidActions) {
-    throw new Error('Action name is a required field');
+    throw new Error(`Invalid Dispatcher: ${dispatcher.name} has actions without a name`);
```

Comment:
These errors point a bit better to where the mistake is and fit in more with the success message.